README-forward-any-tez.md

```diff
@@ -0,0 +1,443 @@
+# Forward-Any-Contract
+
+
+## Originating Contract
+
+
+```bash
+$ tezos-client --wait none originate contract ForwardAnyTezContract transferring 0 from $ALICE_ADDRESS running "$(stack exec -- prototype-forwarder-contract print-specialized-any-tez-fa12 --central-wallet $ALICE_ADDRESS)" --burn-cap 0.645
```

Comment:
please update the readme with the new command name: `print-specialized-any-tez-fa12` -> `print-specialized-forward-any-fa12`